evap/evaluation/models.py

```diff
@@ -812,6 +1083,15 @@ class Meta:
     def is_general(self):
         return self.contributor_id is None
 
+    @property
+    def object_to_attach_logentries_to(self):
+        return Evaluation, self.evaluation_id
+
+    def __str__(self):
+        if self.contributor:
+            return _("Contribution by") + " " + self.contributor.full_name
```

Comment:
Using one f-string here would allow localization to also change word order (which we usually want to allow?)